src/rules/rdf/mustNotExist.js

```diff
@@ -4,7 +4,7 @@ import * as messages from 'messages';
 
 
 export function mustNotExist(xmlDoc, filename=null,
-                             namespace='http://www.mozilla.org/2004/em-rdf#') {
+    {namespace='http://www.mozilla.org/2004/em-rdf#'}={}) {
```

Comment:
I think when I originally coded this it was a bit messy. It should live canonically in the scanner itself and be passed to the rules from the scanners. The rules shouldn't assume anything about the document.

Sorry about that, my fault :astonished: